src/rewards/rewards.service.ts

```diff
@@ -178,6 +203,25 @@ export class RewardsService extends Service {
           this.connector,
         );
       }
+      // TODO: enable once emission control is on the registry
+      // if (emissionControlAddress) {
+      //   this._emissionControl = EmisisonControl__factory.connect(
+      //     emissionControlAddress,
+      //     this.connector,
+      //   );
+      // }
+      if (this.config.network === Network.Ethereum) {
+        this._emissionControl = EmisisonControl__factory.connect(
+          '0x31825c0a6278b89338970e3eb979b05b27faa263',
```

Comment:
can these addresses be somewhere else

Reply:
resolved offline, will be removed once the emission control is on the registry, will submit pr tmrw